packages/lesswrong/server/vulcan-lib/utils.ts

```diff
@@ -61,7 +61,6 @@ export const sanitize = function(s: string): string {
       li: [ 'footnote-item' ],
     },
     allowedStyles: {
-      ...(sanitizeHtml.defaults as any).allowedStyles,
```

Comment:
This was a no-op; see `node_modules/sanitize-html/index.js` for the full `defaults` object, which has no `allowedStyles` prop.